### ISSUE TITLE
hyperparameters different from paper?

### ISSUE THREAD

**Issue by movingright** [harlanhong/CVPR2022-DaGAN#78]
Hi, thanks for the nice repo.

1. Looks like the hyperparmeters you have in [vox-adv-256.yaml](https://github.com/harlanhong/CVPR2022-DaGAN/blob/master/config/vox-adv-256.yaml) are different from what you have on the paper. For example, the loss weight for the discriminator here is 1 (https://github.com/harlanhong/CVPR2022-DaGAN/blob/master/config/vox-adv-256.yaml#L63), but in the 4.2 section of your paper (https://arxiv.org/pdf/2203.06605.pdf) you say  lambda_d = 10.
2. Also, when I was looking to fine tune (for 512p) using your checkpoint on single GPU (batch size 3), taking one (or few <10) generator step(s) with learning rate 2p-4 makes the results much worse (eg, making the frame fully black, making it very blurred, etc.). I had to make the learning rate 2p-5 to not completely screw up things, is this expected? 

Can you confirm that this is expected?

**Comment by movingright** [comment]
Thanks for the quick response. 

The problem with applying GFPGAN on 256p is that high frequency parts of the image (eg. hair) flicker a lot. I was thinking a temporal super resolution algorithm might do better, but can't find any that do well on human faces.
